### ISSUE TITLE
ViewBinding in AppCompatDialogFragment

### ISSUE THREAD

**Issue by enriquebautista** [android/architecture-components-samples#937]
From Official docs:
Fragments outlive their views. Make sure you clean up any references to the binding class instance in the fragment's onDestroyView() method.

`
public class SomeDialogFragment extends AppCompatDialogFragment {

private DialogSomeBinding binding;

@NonNull
    @Override
    public Dialog onCreateDialog(Bundle savedInstanceState) {
binding = DialogSomeBinding.inflate(LayoutInflater.from(requireContext()));
return new MaterialDialog.Builder(requireContext())
                .customView(binding.getRoot(), false)
                .build();
}

// This part is necessary even if you don't use onCreateView?
@Override
    public void onDestroyView() {
        binding = null;
        super.onDestroyView();
    }

}
`

**Comment by amindadgar** [comment]
Yes, Of course you need to clean the memory ( from variable that reference the binding).
**Reason:** Because you instantiate the variable _binding_, by fragment being destroyed you need to free it ( assign it null ).

**Comment by Rishavgupta12345** [comment]
can i work on this issu ?

this is my first contribution
